maprendering/maprendering.dm

```diff
@@ -0,0 +1,109 @@
+/client/proc/maprender()
+	set category = "Mapping"
+	set name = "Generate Map Render"
+
+	if(!holder)
```

Comment:
honestly the check shouldn't exist at all, the only way somebody without the rights could call that is advance proc call, and at that point they already could do so much more damage, including changing their own flags.